packages/core/src/types.ts

```diff
@@ -402,7 +402,7 @@ export type SourceCode = TextSourceCode | BinarySourceCode;
  */
 export interface VisitTraversalStep {
 	kind: 1;
-	target: object;
+	target: unknown;
```

Comment:
Why this change, isn't this expected to be an object (AST node)?

Reply:
`unknown` forces consumers to specify the type of the object before working with it, which ensures that it's properly typed.

`object` will just cause squiggles every time you access a property or method that isn't on `Object.prototype`.